Rubberduck.UnitTesting/UnitTesting/ITestEngine.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Rubberduck.UnitTesting
+{
+    public interface ITestEngine
+    {
+        event EventHandler<TestCompletedEventArgs> TestCompleted;
+        event EventHandler<long> TestRunCompleted;
```

Comment:
`EventHandler<long>` is surprising. Would wrapping it in an `EventArgs`-derived type be overkill? I see below that this is the `ElapsedMilliseconds`; that's rather non-obvious when all you got is a `long` IMO.